lms/envs/common.py

```diff
@@ -3290,6 +3290,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'DEFAULT_RENDERER_CLASSES': (
         'rest_framework.renderers.JSONRenderer',
     ),
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'edx_rest_framework_extensions.auth.jwt.authentication.JwtAuthentication',
```

Comment:
@feanil: When you get a chance, you'll want to rebase (you should have conflicts), and then switch this to:
```
'openedx.core.djangolib.default_auth_classes.DefaultJwtAuthentication',
```

Reply:
Done.